src/styles/index.scss

```diff
@@ -1,3 +1,91 @@
+@import '/src/styles/card';
+@import '/src/styles/variables';
+@import '/src/styles/stars';
+@import '/src/styles/mixin';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here, but it's important to ensure that it is defined in the imported `variables` file. Otherwise, it will cause an error during compilation.